2016/ideas-list-swc.md

```diff
@@ -55,25 +76,23 @@ task, but thanks to their API we can do this automatically in our application.
 
 ### Technical details
 
-Due to the fact that [AMY][] is a [Django][] application, it is suggested:
-
-* to have experience with [Django][] and [Python][]
-* to create the project as [Django application](https://docs.djangoproject.com/en/1.9/ref/applications/#projects-and-applications)
-
-This will allow for faster integration with AMY, if the there's time left in
-the Summer.
+Due to the fact that [AMY][] is a [Django][] application, it is suggested
+to have experience with [Django][] and [Python][]
+since create the project as [Django application](https://docs.djangoproject.com/en/1.9/ref/applications/#projects-and-applications)
+will allow faster integration with AMY and easy maintenance of the code.
```

Comment:
```
Due to the fact that [AMY][] is a [Django][] application, it is suggested
to have experience with both [Python][] and [Django][] since creating
the project as [Django application](https://docs.djangoproject.com/en/1.9/ref/applications/#projects-and-applications)
will allow for faster integration with AMY and easier maintenance in future.
```